instructor_notes/fastq_subset/README.md

```diff
@@ -1,3 +1,3 @@
 These FASTQ files are subsetted versions (to only the first 10,000 reads) of two of the samples from this project: https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP255885.
 
-Files are obtained from EBI and processed with `generate_fastq_subsets.sh`, and they can be used as alternative FASTQ files to download when internet speed is prohibitive.
+Files are obtained from `EBI` and processed with `generate_fastq_subsets.sh`, and they can be used as alternative FASTQ files to download when internet speed is prohibitive.
```

Comment:
EBI is an abbreviation, not code. Also tense change...
```suggestion
Files were obtained from EBI and processed with `generate_fastq_subsets.sh`, and they can be used as alternative FASTQ files to download when internet speed is prohibitive.
```

Reply:
alright, fine.